test/test_user_settings.py

```diff
@@ -34,63 +32,72 @@ class TestUserSettings(unittest.TestCase):
   @classmethod
   def setUpClass(self):
     self.working_dir = os.getcwd()
-    self.test_dir = os.path.realpath(tempfile.mkdtemp())
-    os.chdir(self.test_dir)
 
-    # Backup settings
-    self.settings_backup = {}
-    for key in dir(in_toto.settings):
-      self.settings_backup[key] = getattr(in_toto.settings, key)
-
-    # Create an rc file and set some environment variables
-    with open(".in_totorc", "w") as fobj:
-      fobj.write(
-          "[settings]\n"
-          "artifact_basepath=r/c/file\n"
-          "ARTIFACT_EXCLUDES=r:c:file\n"
-          "new_rc_setting=new rc setting")
+    # We use `demo_files` as test dir because it has an `.in_totorc`, which
```

Comment:
I don't understand this comment.  Is `demo_files` supposed to be `rc_test`?

Reply:
Oh, I forgot to update this comment. I first added the rcfile to demo_files, as you suggested, but then discovered that this would break other tests that copied everything from demo_files and therefor also loaded and used the test rcfile.